server/app/controllers/dev/seeding/SampleQuestionDefinitions.java

```diff
@@ -186,25 +239,35 @@ public final class SampleQuestionDefinitions {
               .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
               .build());
 
-  public static DateQuestionDefinition dateQuestionDefinition(String name, String questionText) {
-    return new DateQuestionDefinition(
-        QuestionDefinitionConfig.builder()
-            .setName(name)
-            .setDescription("description")
-            .setQuestionText(LocalizedStrings.withDefaultValue(questionText))
-            .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
-            .build());
-  }
+  public static final DateQuestionDefinition DATE_QUESTION_DEFINITION =
+      new DateQuestionDefinition(
+          QuestionDefinitionConfig.builder()
+              .setName("Sample Date Question")
+              .setDescription("description")
+              .setQuestionText(LocalizedStrings.withDefaultValue("When is your birthday?"))
+              .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
+              .build());
+
+  public static final DateQuestionDefinition DATE_PREDICATE_QUESTION_DEFINITION =
+      new DateQuestionDefinition(
+          QuestionDefinitionConfig.builder()
+              .setName("Sample Predicate Date Question")
+              .setDescription("description")
+              .setQuestionText(LocalizedStrings.withDefaultValue("When is your birthday?"))
+              .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
+              .build());
```

Comment:
Above are just more clearly declared versions of DateQuestionDefinitions that were previously calling `dateQuestionDefinition(name, text) in `DatabaseSeedController`.